JSDemos/Demos/Autocomplete/Overview/description.md

```diff
@@ -8,21 +8,27 @@ Use one of the following properties to supply data to the component:
 Accepts a local data array.
 
 * [dataSource](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#dataSource)            
-Accepts a local data array, [DataSource](/Documentation/ApiReference/Data_Layer/DataSource/) object, or DevExtreme data store. In this demo, every Autocomplete component is bound to a local array, except the "Custom Item Template and Data Source Usage" component that uses an [ODataStore](/Documentation/ApiReference/Data_Layer/ODataStore/).
+Accepts a local data array, [DataSource](/Documentation/ApiReference/Data_Layer/DataSource/) object, or DevExtreme data store. In this demo, every Autocomplete component is bound to a local array, except the following components:
+
+    - "Custom Item Template and Data Source Usage"    
+    Uses an [ODataStore](/Documentation/ApiReference/Data_Layer/ODataStore/).
+    
+    - "With Custom Store and Search Options"    
+    Uses a [CustomStore](/Documentation/ApiReference/Data_Layer/CustomStore/).
 
 The [value](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#value) property stores the selected item. You can use the same property to select an item programmatically, as shown in the "Disabled" Autocomplete component.
 
-If the data source contains objects, you should specify the [valueExpr](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#valueExpr) property. It accepts a data field name that uniquely identifies each data object. In this demo, **valueExpr** is set only for the "Custom Item Template and Data Source Usage" Autocomplete because others are bound to arrays of primitive values.
+If the data source contains objects, you should specify the [valueExpr](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#valueExpr) property. It accepts a data field name that uniquely identifies each data object. In this demo, [valueExpr](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#valueExpr) is specified only for the "Custom Item Template and Data Source Usage" and "With Custom Store and Search Options" components because others are bound to arrays of primitive values.
```

Comment:
Same note: With Custom... With? 

Also, when you say "others are bound to arrays", what are you referring to? other what? clarify.

Reply:
I can change it to "other components"

Reply:
Yes, other components.